packages/ember-htmlbars/lib/keywords/unbound.js

```diff
@@ -1,8 +1,22 @@
 import Ember from 'ember-metal/core'; // assert
 
 /**
-@module ember
-@submodule ember-htmlbars
+  The `{{unbound}}` helper can be used with bound helper invocations to
+  render them in their unbound form.
+
+  ```handlebars
+  {{unbound capitalize name}}
```

Comment:
Please use the format `{{unbound (capitalize name)}}`. This is the only supported syntax in 2.0.